components/molecule/photoUploader/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "sui-form-image-uploader",
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "npx rimraf ./lib && npx mkdirp ./lib && npm run build:js && npm run build:styles",
+    "build:js": "../../../node_modules/.bin/babel --presets sui ./src --out-dir ./lib",
+    "build:styles": "../../../node_modules/.bin/cpx './src/**/*.scss' ./lib"
+  },
+  "dependencies": {
+    "@s-ui/component-dependencies": "1",
+    "react-dropzone": "10",
```

Comment:
For external dependencies that we don't have any control whatsoever is recommended to fix the exact version to avoid unwanted side effects. Ex:
`"react-dropzone": "10.2.4"`